src/main/java/org/generationcp/middleware/api/germplasm/GermplasmNameServiceImpl.java

```diff
@@ -108,4 +111,9 @@ public Integer createName(final GermplasmNameRequestDto germplasmNameRequestDto,
 		daoFactory.getNameDao().save(name);
 		return name.getNid();
 	}
+
+	@Override
+	public List<GermplasmNameDto> getGermplasmNamesByGids(final List<Integer> gids) {
+		return this.daoFactory.getNameDao().getGermplasmNamesByGids(new ArrayList<>(gids));
```

Comment:
Why instantiate a new array list for the gids parameter?

Reply:
I changed the gids param to Set type before so I had to convert, just forgot to correct this.